.circleci/config.yml

```diff
@@ -958,11 +958,6 @@ jobs:
     machine:
       image: ubuntu-2004-cuda-11.4:202110-01
     resource_class: gpu.nvidia.small
-    environment:
-      PYTHON_VERSION: << parameters.python_version >>
-      PYTORCH_VERSION: << parameters.pytorch_version >>
-      UNICODE_ABI: << parameters.unicode_abi >>
-      CU_VERSION: << parameters.cu_version >>
```

Comment:
Driveby. We don't need these here, because they are pulled in at

https://github.com/pytorch/vision/blob/2b16299fdf6d128ab4cc270f28fef0488489f4e4/.circleci/config.yml#L957

from

https://github.com/pytorch/vision/blob/2b16299fdf6d128ab4cc270f28fef0488489f4e4/.circleci/config.yml#L222-L226